crowdstrike_rules/crowdstrike_dns_request.yml

```diff
@@ -0,0 +1,101 @@
+AnalysisType: rule
+Filename: crowdstrike_dns_request.py
+RuleID: Crowdstrike.DNS.Request
+DisplayName: DNS request to denylisted domain
+Enabled: true
+LogTypes:
+  - Crowdstrike.DNSRequest
+Tags:
+  - Crowdstrike
+Severity: Critical
+Reports:
+  MITRE ATT&CK:
+    - Initial Access:Phishing
+Description: A DNS request was made to a domain on an explicit denylist
+Reference: https://docs.runpanther.io/data-onboarding/supported-logs/crowdstrike#crowdstrike-dnsrequest
+Runbook: Filter for host ID in title in Crowdstrike Host Management console (https://falcon.us-2.crowdstrike.com/hosts/hosts/) to identify the system that queried the domain.
```

Comment:
We should pull the url, some customers are falcon.us-1.crowdstrike.com, etc.

Reply:
Removed